build-system/tasks/e2e/network-logger.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```

Comment:
The Internet told me that we're now closer to 1/1/2050 than we are to 1/1/1990.

Reply:
:open_mouth: